CHANGELOG.md

```diff
@@ -1,19 +1,62 @@
-# 2.1.0 (2017-01-XX)
+# 2.1.0 (2017-02-TBD)
```

Comment:
You know if you leave it this way, you will need to open another PR just to edit the date, right?